src/Symfony/Component/Serializer/Normalizer/DenormalizerInterface.php

```diff
@@ -72,10 +67,11 @@ public function supportsDenormalization(mixed $data, string $type, string $forma
      * returned as keys, and each type should be mapped to a boolean indicating
      * if the result of supportsDenormalization() can be cached or not
      * (a result cannot be cached when it depends on the context or on the data.)
+     * A null value means that the denormalizer does not support the corresponding
+     * type.
      *
-     * The special type '*' can be used to indicate that the denormalizer might
-     * support any types. A null value means that the denormalizer does not support
-     * the corresponding type.
+     * Use type "object" to match any classes or interfaces,
+     * and type "*" to match any types.
      *
      * @return array<class-string|'*'|string, bool|null>
```

Comment:
as we specified '*', should we also specify 'object' to document the special case ?
```suggestion
     * @return array<class-string|'*'|'object'|string, bool|null>
```